packages/algorand-js/test/mocks/txn.ts

```diff
@@ -9,6 +9,9 @@ const GENESIS_HASH = 'SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=';
 const account = generateAccount();
 const addr = decodeAddress(account.addr);
 
+export const addr1 = 'WHVQXVVCQAD7WX3HHFKNVUL3MOANX3BYXXMEEJEJWOZNRXJNTN7LTNPSTY';
+export const addr2 = '2UBZKFR6RCZL7R24ZG327VKPTPJUPFM6WTG7PJG2ZJLU234F5RGXFLTAKA';
```

Comment:
please use `joshnAccount.address` here. 

If we copy things in other places, then update them as well.

Reply:
ok